README.md

```diff
@@ -28,12 +28,35 @@ the reverse proxy is working before you commit. To do this, do the following:
 
 1. [Install Docker][docker]
 
-2. Start the container
+2. Fetch the latest base image:
+    ``` shell
+    $ docker pull nginx
+    ```
+
+3. Optionally change the bound ports so they don't conflict with your local host's
```

Comment:
Maybe a different Rake task would make this easier?

Reply:
Does the `Rakefile` get used in production? If not, I'm tempted to just change the run task.

Reply:
Ok, I'll have a think about how best to do that. I think it's probably best to approach that separately though, as there is already enough info here for people to get going and port 80 already being in use may just be an edge case.